docs/how-to/h-deploy.md

```diff
@@ -1,106 +1,127 @@
 
-## Deploy Charmed Kafka and Charmed ZooKeeper
+Charmed Apache Kafka and Charmed Apache ZooKeeper can both be deployed as follows:
 
-The Kafka and ZooKeeper charms can both be deployed as follows:
-
-```shell
+```commandline
 $ juju deploy kafka --channel 3/stable -n <kafka-units> --trust
 $ juju deploy zookeeper --channel 3/stable -n <zookeeper-units>
 ```
 
-where `<kafka-units>` and `<zookeeper-units>` – the number of units to deploy for Kafka and ZooKeeper. We recommend values of at least `3` and `5` respectively.
+where `<kafka-units>` and `<zookeeper-units>` – the number of units to deploy for Charmed Apache Kafka and Charmed Apache ZooKeeper. We recommend values of at least `3` and `5` respectively.
 
-> **NOTE** The `--trust` option is needed for the Kafka application if NodePort is used. For more information about the trust options usage, see the [Juju documentation](/t/5476#heading--trust-an-application-with-a-credential). 
+[note]
+The `--trust` option is needed for the Apache Kafka application if NodePort is used. For more information about the trust options usage, see the [Juju documentation](/t/5476#heading--trust-an-application-with-a-credential). 
+[/note]
 
-After this, it is necessary to connect them:
+Connect Charmed Apache ZooKeeper and Charmed Apache Kafka by relating/integrating them:
 
 ```shell
 $ juju relate kafka zookeeper
 ```
 
-Once all the units show as `active|idle` in the `juju status` output, the deployment 
-should be ready to be used. 
+Check the status of the deployment:
+
+```commandline
+juju status
+```
+
+The deployment should be complete once all the units show `active` or `idle` status. 
 
 ## (Optional) Create an external admin users
 
-Charmed Kafka aims to follow the _secure by default_ paradigm. As a consequence, after being deployed the Kafka cluster 
+Charmed Apache Kafka aims to follow the _secure by default_ paradigm. As a consequence, after being deployed the Apache Kafka cluster 
 won't expose any external listener. 
 In fact, ports are only opened when client applications are related, also 
-depending on the protocols to be used. Please refer to [this table](/t/charmed-kafka-documentation-reference-listeners/13264) for 
-more information about the available listeners and protocols. 
+depending on the protocols to be used.
 
-It is however generally useful for most of the use-cases to create a first admin user
-to be used to manage the Kafka cluster (either internally or externally). 
+[note]
+For more information about the available listeners and protocols please refer to [this table](/t/13264). 
+[/note]
+
+It is however generally useful for most of the use cases to create a first admin user
+to be used to manage the Apache Kafka cluster (either internally or externally). 
 
 To create an admin user, deploy the [Data Integrator Charm](https://charmhub.io/data-integrator) with 
-`extra-user-roles` set to `admin`
+`extra-user-roles` set to `admin`:
 
-```shell
+```commandline
```

Comment:
**question/non-blocking:** Why `commandline` vs `shell`? I'm sure there was an agreement on it, just curious as to the reasons.

Reply:
I haven't noticed a significant difference in their rendered behaviour so far. As a general rule, we should try to stay consistent, at least inside a single page. That is why I was using a more popular option everywhere I could find. Let me know if you have a preference here.